src/foolscap/test/test_copyable.py

```diff
@@ -266,17 +266,17 @@ def _testCopy4_3(self, why, obj):
         return d
     def _testCopy4_4(self, why):
         why.trap(Violation)
-        self.failUnlessSubstring("token too large", str(why))
+        self.assertIn("token too large", str(why))
 
 class Registration(unittest.TestCase):
     def testRegistration(self):
         rc_classes = copyable.debug_RemoteCopyClasses
         copyable_classes = rc_classes.values()
-        self.failUnless(MyRemoteCopy1 in copyable_classes)
-        self.failUnless(MyRemoteCopy2 in copyable_classes)
-        self.failUnlessIdentical(rc_classes["MyCopyable2name"],
-                                 MyRemoteCopy2)
-        self.failIf(MyRemoteCopy5 in copyable_classes)
+        self.assertTrue(MyRemoteCopy1 in copyable_classes)
+        self.assertTrue(MyRemoteCopy2 in copyable_classes)
+        self.assertIs(rc_classes["MyCopyable2name"],
+                      MyRemoteCopy2)
+        self.assertFalse(MyRemoteCopy5 in copyable_classes)
```

Comment:
Huh.. the hanging indent was preserved here. Manual effort, or inconsistency in the auto-fixer?

Reply:
Again, pls explain what you mean by hanging indent and why it may be a style issue. Thanks.